aries_vcx/Cargo.toml

```diff
@@ -22,6 +22,7 @@ ios-wrapper = ["libvdrtools/ffi_api"]
 modular_dependencies = ["test_utils", "tokio/rt", "tokio/macros"]
 
 [dependencies]
+bs58 = "0.4.0"
```

Comment:
Was about to complain about new dependency but I see this is way superior to the current `rust-base58` :-) 

Created issue to use `bs58` everywhere https://github.com/hyperledger/aries-vcx/issues/694